errai-ioc/src/main/java/org/jboss/errai/ioc/client/api/EnabledByProperty.java

```diff
@@ -43,4 +43,20 @@
    * if the specified system property exists and has the value {@code "true"}.
```

Comment:
We will need to update this javadoc. Now that we can match arbitrary properties, it isn't so much that `negated` matches `false` instead of `true`. A better description would be:

> Normally, the target bean is enabled if the specified system property exists and matches the declared {@code matchValue}. If {@code negated} is {@code true} then this sense is reversed: the bean is disabled if the specified system property equals the {@code matchValue}.